eventsources/sources/resource/start.go

```diff
@@ -250,7 +250,14 @@ func LabelReq(sel v1alpha1.Selector) (*labels.Requirement, error) {
 	if sel.Operation != "" {
 		op = selection.Operator(sel.Operation)
 	}
-	req, err := labels.NewRequirement(sel.Key, op, []string{sel.Value})
+	values := []string{sel.Value}
+	if (op == selection.Exists || op == selection.DoesNotExist) && sel.Value == "" {
```

Comment:
Return an error when `sel.Value != ""`?

Reply:
If operator is exists or doesnotexists, an error will be thrown from labels.NewRequirement if values is not empty.

Should I add an error from our end? @whynowy

Reply:
This is good, thanks!